### PR TITLE
Add CEWE Latest

### PR DESCRIPTION
**Pull request by Cubixzno1**
After making all changes to the cask:

- [x] `brew cask audit --download {{cask_file}}` is error-free.
- [x] `brew cask style --fix {{cask_file}}` reports no offenses.
- [x] The commit message includes the cask’s name and version.
- [x] The submission is for [a stable version](https://github.com/Homebrew/homebrew-cask/blob/master/doc/development/adding_a_cask.md#finding-a-home-for-your-cask) or [documented exception](https://github.com/Homebrew/homebrew-cask/blob/master/doc/development/adding_a_cask.md#but-there-is-no-stable-version).

Additionally, if **adding a new cask**:

- [x] Named the cask according to the [token reference].
- [x] `brew cask install {{cask_file}}` worked successfully.
- [x] `brew cask uninstall {{cask_file}}` worked successfully.
- [x] Checked there are no [open pull requests] for the same cask.
- [x] Checked the cask was not [already refused].
- [x] Checked the cask is submitted to [the correct repo].

[token reference]: https://github.com/Homebrew/homebrew-cask/blob/master/doc/cask_language_reference/token_reference.md
[open pull requests]: https://github.com/Homebrew/homebrew-cask/pulls
[already refused]: https://github.com/Homebrew/homebrew-cask/search?q=is%3Aclosed&type=Issues
[the correct repo]: https://github.com/Homebrew/homebrew-cask/blob/master/doc/development/adding_a_cask.md#finding-a-home-for-your-cask
[version-checksum]: https://github.com/Homebrew/homebrew-cask/blob/master/doc/cask_language_reference/stanzas/sha256.md#updating-the-sha256

### CODE REVIEW

**Comment by Cubixzno1** [comment]
I don't understand Travis is failing..

**Comment by vitorgalvao** [comment]
You had the cask in the wrong directory.

Thank you for the contribution, but I’ll be rejecting this on account of [not being notable enough](https://github.com/Homebrew/homebrew-cask/blob/master/doc/faq/rejected_casks.md).

I doubt a significant portion of our users would use this, and this is a high-maintenance cask with submission errors.

File: Casks/cewe.rb [hunk 1]
```diff
@@ -0,0 +1,12 @@
+cask 'cewe' do
+  version '6.4.4'
+  sha256 '35eed803d6d68e4684063717b285bb7e868080d85d62cb0e9bb05ed883a128f0'
+
+  # photoprintit.com/api/getClient/18299-da_DK/hps was verified as official when first introduced to the cask
```
Comment (vitorgalvao) [inline Casks/cewe.rb]: Example of high-maintenance: needing verification.

File: Casks/cewe.rb [hunk 2]
```diff
@@ -0,0 +1,12 @@
...
+  url 'https://dls.photoprintit.com/api/getClient/18299-da_DK/hps/om_seo_goo_x_18299_x_18299-AF9Ytche8LJh7/mac'
+  name 'CEWE'
+  homepage 'https://www.cewe.dk/'
+
+  app 'cewe.dk.app'
+end
+
```
Comment (vitorgalvao) [inline Casks/cewe.rb]: Extraneous line.

File: Casks/cewe.rb [hunk 3]
```diff
@@ -0,0 +1,12 @@
+cask 'cewe' do
+  version '6.4.4'
```
Comment (vitorgalvao) [inline Casks/cewe.rb]: Seeing as `url` doesn’t point to a specific version and there’s no [appcast](https://github.com/Homebrew/homebrew-cask/blob/master/doc/cask_language_reference/stanzas/appcast.md), this cask needs `version :latest` and `sha256 :no_check`.

**Comment by Cubixzno1** [comment]
@vitorgalvao 

Thanks for the feedback.. But I was planing og using this for a workproject, which would mean a lot of people would use this.. 

Is it possible for me, to use your code "offline", and make it work? 

Best regards.

File: Casks/cewe.rb (hunk 3 above)
Comment (Cubixzno1) [reply Casks/cewe.rb]: I guess I can see your point :)

File: Casks/cewe.rb (hunk 1 above)
Comment (Cubixzno1) [reply Casks/cewe.rb]: How am I supposed to check the file?

Comment (vitorgalvao) [reply Casks/cewe.rb]: Not you, us, hence the high-maintenance.

**Comment by vitorgalvao** [comment]
> Is it possible for me, to use your code "offline", and make it work?

Make a [tap](https://github.com/Homebrew/brew/blob/master/docs/How-to-Create-and-Maintain-a-Tap.md).

**Comment by Cubixzno1** [comment]
Thanks you very much :) !

**Pull request closed without merging**